### DISCUSSION TITLE
Are host and key settings via exe-file name still active?

### DISCUSSION THREAD

**Discussion by vavilov** [rustdesk/rustdesk#1246] [Q&A]
Hello and thanks for such great remote access software.

I tried to set self-hosted server address via .exe naming like this (domain name hidden by * here):
`rustdesk-host=rustdesk.********.**,key=XvT7JJGTi1d1Immu7Cd3bu83fefJk0F0WgqL5JwKCEc=.exe`
It seems to be ok while looking to manual and source code around license.rs but doesn't work for me on several different Windows machines

**Comment by tetrapac74** [comment]
I also have the same problem on Windows 10 21H2 (Build 19044.2130) and Windows 11 22H2 (Build 22621.674).

**Comment by ibapt53** [comment]
I have the same problem. 

I think it is the last character, the "=" is not taken into account. When you delete it, it works. 

I don't know how to generate a key without the "=" at the end.

**Reply by ghost** [reply]
You simple can't for now, it'll be implemented in future server version

**Comment by rustdesk** [comment]
Nobody put a fix following this PR https://github.com/rustdesk/rustdesk/pull/2058/files/e1516d809f9494db123487a02556ffbf8a27b7ef..29978f1a3eab967d3be9a491b54965167402c104

https://github.com/rustdesk/rustdesk/blob/c1af464203a472b439b648199e75a375ff52b194/libs/hbb_common/src/config.rs#L902

at here https://github.com/rustdesk/rustdesk-server/blob/848b5aedb7e8bda0f582706c792ffbab2d657a39/src/common.rs#L121?
